geomstats/geometry/discrete_surfaces.py

```diff
@@ -514,26 +514,23 @@ def __init__(self, space, a0=1.0, a1=1.0, b1=1.0, c1=1.0, d1=1.0, a2=1.0):
         self.d1 = d1
         self.a2 = a2
 
-        self.exp_solver = DiscreteSurfacesExpSolver(space, n_steps=10)
+        if not gs.__name__.endswith("numpy"):
```

Comment:
Isn't there a less hacky way to test the backend?

Reply:
for the name no... probably something to add.